duckduckgo/iOS
# Pick App Icon during onboarding

Task/Issue URL: https://app.asana.com/0/856498667320406/1161590609052529
Tech Design URL:
CC:

**Description**:
Onboarding experiment: Added screen to present user an option to choose App Icon during onboarding.

Note: all non-supported devices (e.g. iOS <10.3) are removed from both experiment and control group pool.

**Steps to test this PR**:

Control:
1. Run onboarding.
2. Only welcome and home row screens should be shown.
3. Check if App Icon picker in settings works as expected.

Experiment:
1. Run onboarding.
2. App Icon picker should be presented as a 2nd screen.
3. Check if App Icon picker in settings works as expected.

Check iPhone SE / 11 / 11 Pro Max & iPad layout.

Non supported devices:
1. Change `isAppIconChangeSupported` so it return false.
2. Reinstall the app - no variant should be allocated.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
